### PR TITLE
BigAnimal - PEMx updates

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Added node-level subtopic

### CODE REVIEW

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx
Comment (YogeshJain96) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx:1]: pls also update the directory name `monitoring_using_pemx`, as the url shows the directory name in path

Screenshot 2024-03-05 at 5 26 56 PM

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx [hunk 1]
```diff
@@ -19,48 +19,55 @@ To see more in-depth metrics specific to the Postgres cluster, select any widget
...
-  - **Database size** (text value) &mdash; The amount of storage volume used by the Postgres cluster.
+  - **Disk IOPS** (gauge chart) &mdash; The disk I/O operations ratio of the total disk capacity per second for the Postgres cluster.
 
 ## Historical charts
 
 By default, these charts displays the historical data of the last 15 minutes. To view the historical data of a particular time range, customize the time range using a time-range picker. These charts display key metrics in single-line and bar or multi-line and bar form. They provide a concise snapshot of the information such as:
   - **Memory** (line chart) &mdash; The historical trend of memory usage percentage over a time period.
   - **CPU** (bar chart) &mdash; The historical trend of CPU usage percentage over a time period.
-  - ** Network activity** (bar chart) &mdash; The historical data transfer to and from the network card per second, over a time period.
+  - **Network activity** (bar chart) &mdash; The historical data transfer to and from the network card per second, over a time period.
```
Comment (gabrosys) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx:52]: I think that we can update the Network activity as a (multi-line chart), thanks

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx
Comment (aviola) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx:1]: Please also update the errors list in `Charts error state` section by adding the one for the `Disk IOPS` single-value chart. In the solution column, you may write something like "On the Edit Cluster page, go to the cluster settings, volume properties and increase the IOPS within the allowed range".

Comment (aviola) [reply product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx:1]: Is the "Error" column meant to show **exactly** the message that will be displayed in the UI? If so, here you can find the actual messages as we updated them recently:

```
'CPU usage on the cluster has reached {value}%, exceeding the 80% threshold'
'Memory usage on the cluster has reached {value}%, exceeding the 80% threshold'
'Storage usage on the cluster has reached {value}%, exceeding the 80% threshold'
'Disk IOPS usage on the cluster has reached {value}%, exceeding the 80% threshold'
```

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/index.mdx [hunk 2]
```diff
@@ -8,7 +8,7 @@ Monitoring integrations are configured at the project level in BigAnimal. You ca
 
 By default, all the integrations are disabled. After creating the project, enable an integration using the **Integrations** tab.
 
-All the metrics collected from all the clusters in the project are sent to the integrated tool and displayed in the BigAnimal  [**Monitoring and logging** tab using PEMx](../monitoring_using_pemx). The collected logs are exported to the object storage by default.
+All the metrics collected from all the clusters in the project are sent to the integrated tool and displayed in the BigAnimal  [**Monitoring and logging** tab using BigAnimal's Observability](../monitoring_using_pemx). The collected logs are exported to the object storage by default.
```
Comment (YogeshJain96) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/index.mdx:11]: please update the hyperlink accordingly

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_biganimal_observability/index.mdx [hunk 3]
```diff
@@ -19,48 +22,55 @@ To see more in-depth metrics specific to the Postgres cluster, select any widget
 
 To view the **Monitoring and Logging** tab from the BigAnimal portal:
 
-1.  In the left navigation of BigAnimal portal, go to **Clusters**. 
+1. In the left navigation of BigAnimal portal, go to **Clusters**. 
 2. Select any ready cluster. 
 3. On the cluster detail page, select the **Monitoring and Logging** tab.
 
-The **Monitoring and Logging** tab displays the detailed PEMx monitoring metrics specific to that cluster in the form of charts:
+The **Monitoring and Logging** tab displays the detailed monitoring metrics at [different levels](#different-levels) in the form of charts:
```
Comment (sonotley) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_biganimal_observability/index.mdx:29]:
```suggestion
The **Monitoring and Logging** tab displays the detailed monitoring metrics at a [user-specified level of aggregation](#levels-of-aggregation) in the form of charts:
```

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_biganimal_observability/index.mdx [hunk 4]
```diff
@@ -19,48 +22,55 @@ To see more in-depth metrics specific to the Postgres cluster, select any widget
...
 
 - [Single-value charts](#single-value-charts)
 - [Historical charts](#historical-charts)
 
+## Different levels
```
Comment (sonotley) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_biganimal_observability/index.mdx:34]:
```suggestion
## Levels of aggregation
```

**Review by sonotley — APPROVED** [review]
Looks good to me on a quick read - suggested alternative title for 'different levels' as requested.

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/index.mdx [hunk 5]
```diff
@@ -20,7 +20,7 @@ With BigAnimal, you have a few options for monitoring and logging solutions:
 
 - BigAnimal provides a Prometheus-compatible endpoint you can use to connect to your own metrics infrastructure, such as your AWS Managed Grafana. It also provides the option to view logs from your cloud provider's blob storage solution. For more information, see [Other monitoring and logging solutions](other_monitoring). This ability is an optional solution when you're using your own cloud account but is the only option when using BigAnimal's cloud account.
 
-- BigAnimal provides PEMx as another metric endpoint. PEMx consolidates and exposes metrics in each data plane. PEMx is an integrated monitoring and alerting solution in BigAnimal. It's designed to monitor performance of the Postgres clusters. See [PEMx monitoring](monitoring_using_pemx) for more information.
+- BigAnimal provides BigAnimal Observability as another metric endpoint. It consolidates and exposes metrics in each data plane. It is an integrated monitoring and alerting solution in BigAnimal. It's designed to monitor performance of the Postgres clusters. See [BigAnimal Observability](monitoring_using_pemx) for more information.
```
Comment (YogeshJain96) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/index.mdx:23]: Pls update the hyperlink here as well^

**Pull request merged**